docs/attributes-registry/profile.md

```diff
@@ -0,0 +1,51 @@
+<!--- Hugo front matter used to generate the website version of this page:
+--->
+
+<!-- NOTE: THIS FILE IS AUTOGENERATED. DO NOT EDIT BY HAND. -->
+<!-- see templates/registry/markdown/attribute_namespace.md.j2 -->
+
+# Profile
+
+- [Profile Build Id](#profile-build-id-attributes)
+- [Profile Frame](#profile-frame-attributes)
+- [Profile Link](#profile-link-attributes)
+
+## Profile Build Id Attributes
+
+Unique identifier for an executable.
+
+| Attribute              | Type   | Description                                                               | Examples                                                                              | Stability                                                        |
+| ---------------------- | ------ | ------------------------------------------------------------------------- | ------------------------------------------------------------------------------------- | ---------------------------------------------------------------- |
+| `profile.build_id.gnu` | string | The GNU build ID as can be found in the `.note.gnu.build-id` ELF section. | `c89b11207f6479603b0d49bf291c092c2b719293`                                            | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `profile.build_id.go`  | string | The Go build ID.                                                          | `foh3mEXu7BLZjsN9pOwG/kATcXlYVCDEFouRMQed_/WwRFB1hPo9LBkekthSPG/x8hMC8emW2cCjXD0_1aY` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
For the record, it looks like Go build ID is not handled correctly in Go's runtime/pprof and pprof CLI itself. I filed https://github.com/golang/go/issues/68652.

@felixge FYI